File: backend/src/main/java/com/okatakese/stock_app/model/StockDataModel.java
package com.okatakese.stock_app.model;

import lombok.Data;

@Data
public class StockDataModel {
    /** 銘柄コード情報 */
    private TickerDataModel tickerData;
    /** 配当金情報 */
    private DividendDataModel dividendData;
}

File: frontend/src/model/StockDataModel.ts
import type { DividendDataModel } from "./DividendDataModel";
import type { TickerDataModel } from "./TickerDataModel";

export interface StockDataModel {
    /** 銘柄コード情報 */
    tickerData: TickerDataModel;
    /** 配当金情報 */
    dividendData: DividendDataModel;
}